interview-questions.md

```diff
@@ -5,54 +5,55 @@ First, without external resources. Challenge yourself to answer from memory.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.  
 
```

Comment:
Need a bit more content in your answers. Recommend to include these three things:

1. writing the definition
2. describe how you use it in class
3. research something interesting about it